book/src/building/swift-packages/README.md

```diff
@@ -238,3 +233,38 @@ cd SwiftProject
 swift run
 # You should see "Hello from Rust!" in your terminal.
 ```
+
+# Usage on Linux
+Usage on Linux is a little bit different than on macOS. This guide will help you set up SwiftPM and Cargo. The general workflow is:
+* Set up Package.swift so that "cargo build" is called (You can use something like this here: "https://stackoverflow.com/questions/26971240/how-do-i-run-a-terminal-command-in-a-swift-script-e-g-xcodebuild" to invoke cargo during a swift build)
+* Set up swift-bridge according to the guide (see chapter "swiftc and cargo")
```

Comment:
Set up `swift-bridge` how? What does the user need to do? Everything in the guide? A couple things in the guide? Can be more explicit here.